smtp/parser.go

```diff
@@ -37,27 +37,34 @@ func (p *parser) ParseCommand(br *bufio.Reader) (command Cmd, err error) {
 
 	case "HELO":
 		{
-			// TODO: < 1 or == 1?
```

Comment:
`== 1` is better... (it strictly follows the RFC)  
Although this could be taken care of by adding a "strict" configuration flag.